CMakeLists.txt

```diff
@@ -119,22 +86,6 @@ if(UR_ENABLE_TRACING)
     endif()
 endif()
 
-if(UR_USE_ASAN)
-    add_sanitizer_flag(address)
-endif()
-
-if(UR_USE_UBSAN)
-    add_sanitizer_flag(undefined)
-endif()
-
-if(UR_USE_MSAN)
-    message(WARNING "MemorySanitizer requires that all code is built with
-        its instrumentation, otherwise false positives are possible.
-        See https://github.com/google/sanitizers/wiki/MemorySanitizerLibcxxHowTo#instrumented-libc
-        for details")
-    add_sanitizer_flag(memory)
-endif()
-
```

Comment:
I'd argue that we _do_ want third party libraries compiled with sanitizer flags, sanitizers work best when all code has the instrumentation compiled in.